roxctl/connectivity-map/netpol_test.go

```diff
@@ -163,6 +169,33 @@ func (d *analyzeNetpolTestSuite) TestAnalyzeNetpol() {
 			expectedAnalysisError: nil,
 			outputToFile:          true,
 			outputFormat:          "json",
+			expectedOutput: `[
```

Comment:
nit(applies to the other expected output as well): could make this a file within `testdata/...`, but it's not strictly required (I'm not sure how many other test cases and expected outputs we're going to have, if we foresee that it's going to be a few more then we probably should move to `testdata` IMO.

Reply:
Moved